### DISCUSSION TITLE
Links to Google Collab are not working

### DISCUSSION THREAD

**Discussion by alexheat** [community/community#7757] [Other Feature Feedback, Questions, & Ideas] [answered]
Throughout GitHub links to Goggle Colab are not working. They result in an HTML error "This content is blocked. Contact the site owner to fix the issue."

For example see this Gist https://gist.github.com/alexheat/782cbebc117d821b42baa12a4b3811d4. There is a link to open in Google Colab automatically inserted by GitHub but it no longer works. 

And if you manually add such a link to a notebook it no longer works. See https://gist.github.com/alexheat/782cbebc117d821b42baa12a4b3811d4

Here is a bit of HTML that used to work for these kind of links but no longer works. 

`<a href="https://colab.research.google.com/github//pylabel-project/samples/blob/main/label_new_dataset.ipynb" target="_blank"><img src="https://colab.research.google.com/assets/colab-badge.svg" alt="Open In Colab"/></a> `

**Comment by longavailable** [comment]
I ran into the same issue.

**Comment by Adina77** [comment]
Links to Colab not working for me either, but the Colab button on GitHub is just dead. It is recognized as a button but nothing happens when I click. I do not get any error messages. This had been working so nicely before. I can still save to GitHub from Colab, and specify to add the link. So, saving works. But the link does not work, which is a  problem when collaborating in a group. It broke sometime between Nov 12 (the last time I used it successfully) and Nov 16 (the next time I tried).

**Reply by jmalbornoz** [reply]
Same here but with a twist: the "Open in Colab" button normally shows up at the top of the notebook, right in the middle. Now the button is on the left - when i click on it I also get the "This content is blocked. Contact the site owner to fix the issue." message.

**Reply by Adina77** [reply]
Yes, I noticed that too -- the button moved from center to the left. But for me it is just dead. I do not get an Access Denied message but the button no longer works. One day before my group has a project deadline :-(
But I wonder if the issue is with GitHub or Colab/Google.

**Reply by alexheat** [reply]
It is not a Google Colab issue because you would get the same error no matter what URL you link to. It is something with the link handling on Github, but we all notice it on the Google Colab links because it is such a common link to use on a notebook.

**Comment by angelv-salazar** [comment]
Ir does not work from the github pague but you can still open ir from the
Google colab page, when import from github

**Comment by alexheat** [comment, marked as answer]
This is working again everyone!
